crates/oxc_linter/src/rules/import/first.rs

```diff
@@ -0,0 +1,194 @@
+use oxc_ast::{ast::Statement, AstKind};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule};
+
+fn first_diagnostic(span: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn("Using expressions between imports are prohibited")
```

Comment:
Maybe "expressions" should be changed to something like "statements" or something else?

Reply:
Yeah the messages in https://github.com/import-js/eslint-plugin-import/blob/main/src/rules/first.js is also not good.

The message should reflect where the span is pointing at.

Something like "import statements must come first"?